### PR TITLE
add getComments and description to the endpoint

### PR DESCRIPTION
**Pull request by karobarari**

### CODE REVIEW

File: app/app.controllers.js [hunk 1]
```diff
@@ -16,27 +17,36 @@ exports.apiDescription = (req, res, next) => {
...
+      res.status(200).send(article);
+    })
+    .catch(next);
+};
+
+exports.getCommentsByArticle = (req, res, next) => {
+  const { article_id } = req.params;
+  selectComment(article_id)
+    .then((article) => {
+      res.status(200).send(article);
```
Comment (Keb84) [inline app/app.controllers.js:48]: We should be sending our response back in an object 👀  - what are we sending back? Would this be more semantic if it was on a key of comments?

File: app/app.models.js [hunk 2]
```diff
@@ -72,10 +68,27 @@ exports.selectArticlesById = (article_id) => {
...
   });
 };
 
+exports.selectComment = (article_id) => {
+  queryString = `SELECT * FROM comments`;
+  queryValue = [];
+
+  if (article_id) {
+    queryString += ` WHERE article_id = ${article_id} 
+    ORDER BY created_at ASC
```
Comment (Keb84) [inline app/app.models.js:84]: do you want the created_at ascending by default?

**Review by Keb84 — CHANGES_REQUESTED** [review]
Nice work so far. Please check my comments and resubmit 🙂

File: app/app.models.js (hunk 2 above)
Comment (karobarari) [reply app/app.models.js:84]: thanks ;)

Comment (karobarari) [reply app/app.models.js:84]: :)*

**Pull request merged**